aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -92,6 +93,11 @@ export const SignUpExample = () => {
       const adjustedRule = { ...rule };
       const valid = adjustedRule.regexp.test(values.password);
       adjustedRule.valid = valid;
+      if (valid !== true) {
+        setBackgroundError('#FC61613D');
```

Comment:
Could we reference the theme value here?

Reply:
Also, can you also summarize why the grommet validation isn't applying properly for the background color again? I know we discussed, but I forgot the details and it would be good to have documented in case it could be a future grommet unhancement.